### DISCUSSION TITLE
Log drop issue in Tail plugin 1.7.1 version

### DISCUSSION THREAD

**Discussion by Yadumathur** [fluent/fluent-bit#4350] [Q&A] [answered]
We have a K8 set up where the fluent bit is running as DS, we are seeing a log drop issue, and later on debugging we did observe the logs getting dropped just after INPUT, we did STDOUT after INPUT and validated the number of events on both ends.

We are using 1.7.1 version and our INPUT config is as below 

[INPUT]
Name                         tail 
Path                           /var/log/containers/*.log
Tag                             kube.*
Parser                         docker
Skip_long_lines          true
Docker_Mode              On
Refresh_Interval          2
Ignore_older                1s
Rotate_Wait                 5s
Buffer_Chunk_Size     10k
Buffer_Max_Size         15k
Mem_Buf_Limit           5G

We could replicate the issue by increasing the load and seeing a drop of 50% and more just after tail but when we revert 1.6.10 we do not see the issue as much. There is still a log loss but which can handled better by changing buffer sizes.
I can see a bug on Ignore_older not working as expected #3873 in tail plugin which is kind of working in 1.6.10 

We have a theory on high load this could be an issue with less buffer size ? Is this a known issue or any findings which I can look into ?

**Comment by agup006** [comment]
1.7.1 is quite old, do you see the same in 1.8.10?

**Comment by edsiper** [comment, marked as answer]
Closing discussion as no feedback has been provided. Please upgrade to the latest v1.8 series.
